### PR TITLE
quotes added to avoid error

### PR DESCRIPTION
**Pull request by ckavili**
#### What is this PR About?
I was getting the following error when I try to `helm template..`

```
Error: YAML parse error on cronjob-ldap-group-sync/templates/cronjob.yaml: error converting YAML to JSON: yaml: line 42: found character that cannot start any token
```

The quotes are there to avoid this.

#### How do we test this?
`helm template -f values.yaml . `

cc: @redhat-cop/casl

### CODE REVIEW

File: charts/cronjob-ldap-group-sync/templates/cronjob.yaml [hunk 1]
```diff
@@ -39,5 +39,5 @@ spec:
           - name: ldap-bind-password
             secret:
               secretName: {{ .Values.ldap_bind_password_secret }}
-  schedule: {{ .Values.schedule }}
+  schedule: '{{ .Values.schedule }}'
```
Comment (pabrahamsson) [inline charts/cronjob-ldap-group-sync/templates/cronjob.yaml]: What do you think of `{{ .Values.schedule | quote }}` to match with upstream k8s[0] and use a `"` string?

[0] https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#example

**Review by pabrahamsson — COMMENTED** [review]
Thank you @ckavili 
I left a question inline, let me know what you think/prefer.

File: charts/cronjob-ldap-group-sync/templates/cronjob.yaml (hunk 1 above)
Comment (ckavili) [reply charts/cronjob-ldap-group-sync/templates/cronjob.yaml]: haha, of course, that'd be way better 🙈 Thanks!

**Review by pabrahamsson — APPROVED** [review]
Thanks for the fix @ckavili LGTM

**Pull request merged**